### PR TITLE
[KEP-2413] Add docs for SeccompDefault graduation

### PR DESCRIPTION
**Pull request by saschagrunert**
PR for https://github.com/kubernetes/enhancements/issues/2413

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] [KEP-2413] Add docs for SeccompDefault graduation

**Comment by mickeyboxell** [comment]
Hi @saschagrunert 👋 please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review as soon as possible. 01:00 UTC Wednesday 22nd March 2023 / 17:00 PDT Tuesday 21st March 2023 is the official deadline.

This PR will need a doc review by Tuesday 4th April 2023 to get this into the release. Please reach out to required SIGs to get their review. Thank you!

**Comment by saschagrunert** [comment]
Ready for review.

File: content/en/docs/tutorials/security/seccomp.md [hunk 1]
```diff
@@ -272,7 +265,7 @@ or not. You can adopt these defaults for your workload by setting the seccomp
 type in the security context of a pod or container to `RuntimeDefault`.
 
 {{< note >}}
-If you have the `SeccompDefault` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+If you have the `SeccompDefault` [configuration](/docs/reference/command-line-tools-reference/kubelet)
```
Comment (tengqm) [inline content/en/docs/tutorials/security/seccomp.md:268]:
```suggestion
If you have the `seccompDefault` [configuration](/docs/reference/config-api/kubelet-config.v1beta1/)
```

Comment (saschagrunert) [reply content/en/docs/tutorials/security/seccomp.md]: Thank you! Fixed as suggested :+1:

**Comment by saschagrunert** [comment]
Had to rebase this one.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -308,6 +308,9 @@ For a reference to old feature gates that are removed, please refer to
 | `RemoveSelfLink` | `false` | Alpha | 1.16 | 1.19 |
 | `RemoveSelfLink` | `true` | Beta | 1.20 | 1.23 |
 | `RemoveSelfLink` | `true` | GA | 1.24 | - |
+| `SeccompDefault` | `false` | Alpha | 1.22 | 1.24 |
+| `SeccompDefault` | `true` | Beta | 1.25 | 1.26 |
+| `SeccompDefault` | `true` | GA | 1.27 | - |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:313]: line 182-183 should be removed?

Comment (saschagrunert) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:311]: Good catch, fixed as suggested :+1:

**Pull request merged**